packages/python-packages/apiview-gpt/src/_python_api.py

```diff
@@ -0,0 +1,6 @@
+from ._gpt_reviewer import GptReviewer
+
+def review_python(code):
```

Comment:
This should really accept code and language, as apiview-gpt should be able to work with any language. APIView (web) would be responsible for passing the appropriate language.

Reply:
In the plan, this file will contain all Python related services. e.g. BreakingChangeDetector. And .net, js, java will have their own _language_api.py.